cmd/stack.go

```diff
@@ -82,7 +84,7 @@ func setupStackCommand() *cobra.Command {
 	}
 	upCommand.Flags().BoolP(cobraext.DaemonModeFlagName, "d", false, cobraext.DaemonModeFlagDescription)
 	upCommand.Flags().StringSliceP(cobraext.StackServicesFlagName, "s", nil,
-		fmt.Sprintf(cobraext.StackServicesFlagDescription, strings.Join(availableServicesAsList(), ", ")))
+		fmt.Sprintf(cobraext.StackServicesFlagDescription, fmt.Sprintf("\"%s\"", strings.Join(availableServicesAsList(), ","))))
```

Comment:
Nit: to prevent needing escaping the inner double quote.
```suggestion
		fmt.Sprintf(cobraext.StackServicesFlagDescription, fmt.Sprintf(`"%s"`, strings.Join(availableServicesAsList(), ","))))
```

Reply:
Or you could just avoid the `fmt.Sprintf` altogether.
```suggestion
		fmt.Sprintf(cobraext.StackServicesFlagDescription, `"` + strings.Join(availableServicesAsList(), ",") + `"`))
```